### DISCUSSION TITLE
How does the fuel train work with the fuel supply station?

### DISCUSSION THREAD

**Discussion by ShaneYu** [Opinionated-Blueprints/10-Books-Full-of-Rails#19] [Q&A] [answered]
I have setup a fuel supply station and added a fuel trail like this:
![image](https://github.com/user-attachments/assets/3c6dbe9d-6b20-4661-a367-72ebaacd0514)

I have then setup a fueling station like this:
![image](https://github.com/user-attachments/assets/60a531bf-7b6b-445e-8b93-c61235489a62)

However the fuel train just sits there an nothing is ever loaded into the train cargo waggon, have I don this incorrectly?
I have seen it load nuclear fuel into the wagon once but then was immediately removed again; so the top box is empty, there is fuel sitting on the input / buffer belt and the box below is mixed of all types of fuels as I've tried putting all of them through.

This is what the fuel train looks like after I place it freshly:
![image](https://github.com/user-attachments/assets/8e4e4f56-29b8-4fd3-a9e3-1a5f14e28ce4)

----

I love these train blueprints, probably the best ones out there, however I am very confused when it comes to the fuel site. I think I guide (perhaps video) to show how you use the blueprints would be very handy and explain things little better via examples.

**Comment by Bocian-1** [comment, marked as answer]
https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/wiki/Refuelling

**Reply by ShaneYu** [reply]
Thanks, I've already gone through all of that so I was confused why the fuel train just sites there and never moves; so I didn't know if there were and issue with the BP or I've done something wrong.

I'll give it another go this evening.

**Comment by Bocian-1** [comment]
Alt mode would be useful for the future. Those pictures don't tell me anything without it.

**Reply by ShaneYu** [reply]
I gave it another go and it appears to be working at the moment. Not tried using a higher tier fuel yet, but it's working.

I must have messed something up somewhere, thanks.
